[POST] pyOpenSci Fall Festival event page (#462)
pyOpenSci/pyopensci.github.io: _posts/events/2024-08-09-pyopensci-fall-festival.md

comments: true
---

## <i class="fa-regular fa-heart"></i> Announcing the pyOpenSci Fall Festival!

> _This post last updated on 6 August, 2024_

Reviewer: ```suggestion
_This post was last updated on 6 August, 2024_
```

> Happening October 28--November 01, 2024, the pyOpenSci Fall Festival is an online community training and networking event designed to empower scientists with in demand open science skills. Our goals with the Fall Festival are to empower the community with technically-relevant open science skills, build community, and help raise funds for pyOpenSci operations that allow us to continue supporting community work. Interested? Read on to learn more!

Reviewer: ```suggestion
Happening October 28--November 01, 2024, the pyOpenSci Fall Festival is an online community training and networking event designed to empower scientists with in demand open science skills. Our goals for the Fall Festival are to 

* empower you with technically-relevant open science skills, 
* call attention to and celebrate new and upcoming tools that support open reproducible science
* build community, and 
* help raise funds for pyOpenSci operations that allow us to continue supporting community work. 

Interested? Read on to learn more!
```

Reviewer: i switched it to "you" - only because it had the word "community" so many times i wasn't sure who we were speaking too. The bigger question around fundraising as a goal. i love the transparency. do other organizations write that in their event pages? like psf and pycon? maybe they do! i just don't know what is best.

## <i class="fa-regular fa-file-lines"></i> pyOpenSci's 2024 Fall Festival logistics:

* **What:** An online community training and networking event
* **Where:** Online using [Gather](https://www.gather.town/)
* **When:** Monday, October 28--Friday, November 01, 2024

> * **Who:** Anyone with basic Python skills working, studying or interested in open science

Reviewer: ```suggestion
* **Who:** If you are a scientist, a researcher, a student or anyone who is interested in making your workflows better, more open and reproducible, then this event is for you!   
```

> * **Cost:** $350--$600, with scholarships and learner sponsorship opportunities

Reviewer: ```suggestion
* **Cost:** $300--$600, with scholarships and learner sponsorship opportunities
```

> * Resources:
>   *  [**Workshop GitHub repository:**](https://github.com/pyOpenSci/code-to-module-workshop/)
>   *  [**pyOpenSci demo package:**](https://github.com/pyOpenSci/pyosPackage)

Reviewer: ```suggestion
* [**pyOpenSci demo package:**](https://github.com/pyOpenSci/pyosPackage)
```
 maybe instead link to the hatch tutorials as a second just for the time being?

## pyOpenSci's Fall Festival registration, cost, and scholarship information

### Registration

**Register for the pyOpenSci Fall Festival:** _stay tuned! Registration will be going live soon!_  
Registration will grant you admission to all 

### Pricing tiers

In order to make the Fall Festival accessible to as many people as possible, we've created several pricing tiers. 

> * Students: $300
> * Academics: $500
> * Corporate: $600

Reviewer: ```suggestion
* Student: $300
* Standard: $450
* Corporate: $600
```

### Scholarships

If you are financially unable to attend pyOpenSci's Fall Festival, please reach out to us at [media@pyopensci.org](mailto:pyopensci.org) to request a link to our scholarship application. We will be accepting applications until Friday, September 20th, and sending out notifications the week of September 30th, 2024.

## pyOpenSci Fall Festival events

Our Fall Festival is broken down into a series of events that make it easy to connect with other people, listen to inspiring talks, and learn at a comfortable pace. 

### Monday, October 28th

We're excited to share two events happening on Monday, October 28th. The first will be a kick-off talk from an incredible speaker. This Fall Festival kickoff talk will be free, open to the public, recorded, and later published to YouTube.  

> We'll also be holding "Day 0" office hours for all registered attendees. This will be a block of time that can be used as drop-in office hours, allowing you to say hello and get any assistance you might need with setting up your computer for the upcoming workshops.

Reviewer: ```suggestion
We'll also be holding "Day 0" office hours for all registered attendees. You can use this time to drop in to say hello, get used to using our online platform, gather and get help with any computer setup issues that you might have. 
```
